src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -23,19 +23,18 @@ public class AddCommand extends Command {
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a person to the address book.\n"
             + "Parameters: "
-            + PREFIX_NAME + "NAME "
-            + PREFIX_EMAIL + "EMAIL "
-            + PREFIX_GENDER + "GENDER "
-            + PREFIX_AGE + "AGE "
-            + "[" + PREFIX_DETAIL + "DETAIL] "
-            + "[" + PREFIX_STUDY_GROUP_TAG + "STUDY-GROUP-TAG]..."
-            + "\n"
+            + PREFIX_NAME + "NAME\n"
```

Comment:
Should we standardise how the params are displayed? Currently, `Messages::format` adds a space at the start of a new line.

Reply:
Thx for pointing it out, a space is now added before params.